src/styles/themes/libris.css

```
@@ -753,14 +769,33 @@ markup), and the shorthand would reset it out from under them. */
         --bevel-fill: url("/noiseTexture-midcontrast-desaturated.webp");
         --bevel-fill-blend: soft-light;
         background-color: var(--color-base-300);
-        z-index: 1;
+        z-index: 4;
         translate: -0.4em 0;
         color: var(--color-base-content);
         padding-right: 2.2em;
         text-shadow: 0 0 1em white;
         /*filter: drop-shadow(0 0 1em white);*/
```

Stylelint (17.14.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)